sven-co-opconfig.json

```diff
@@ -12,6 +12,21 @@
         "DefaultValue": "AMP Powered Sven Server",
         "EnumValues": {}
     },
+    {
+        "DisplayName": "Max Players",
+        "Category": "Server Settings",
+        "Description": "maxplayers",
+        "Keywords": "maxplayers,limit",
+        "FieldName": "$MaxUsers",
+        "InputType": "number",
+        "MinValue": "1",
```

Comment:
It seems the MinValue should be 2, and can you please add a MaxValue of 32?

Reply:
Done it for you, lazy

Reply:
I was wanting feedback since I've never played this game. Just going by a guide. And as you know, mods add changes to games that could change the max value for things like this. Wanted to avoid that scenario.

Reply:
Well, that’s fair.

Reply:
I'm not finding any proof that there's more than 32 allowed even with mods. So lets go ahead and close this one. Thanks for updating it!

Reply:
Yeah max is 32 regardless of mods. Engine limit

Reply:
Awesome. Thanks for confirming!